### PR TITLE
make cost be tracked in 64 bit variable instead of 32

### PR DESCRIPTION
**Pull request by arvidn**

### CODE REVIEW

File: src/more_ops.rs [hunk 1]
```diff
@@ -17,59 +18,59 @@ use crate::serialize::node_to_bytes;
...
 
-const PUBKEY_BASE_COST: u32 = 394;
-const PUBKEY_COST_PER_BYTE_DIVIDER: u32 = 4;
+const PUBKEY_BASE_COST: Cost = 394;
+const PUBKEY_COST_PER_BYTE_DIVIDER: Cost = 4;
 
-fn limbs_for_int(v: &Number) -> u32 {
-    ((v.bits() + 7) >> 3) as u32
+fn limbs_for_int(v: &Number) -> usize {
+    ((v.bits() + 7) / 8) as usize
```
Comment (richardkiss) [inline src/more_ops.rs:73]: It's a matter of taste :)

Comment (arvidn) [reply src/more_ops.rs:73]: I saw you changed it in the other PR, so I figured I'd do it here too.. or did I misread that?

Comment (richardkiss) [reply src/more_ops.rs:73]: No, but I did go to a rust compiler explorer to confirm that rust would compile them the same. It's exactly the same for `usize`, but very slightly different for `i32`. [Check it out](https://rust.godbolt.org/#g:!((g:!((g:!((h:codeEditor,i:(fontScale:14,j:1,lang:rust,selection:(endColumn:1,endLineNumber:18,positionColumn:1,positionLineNumber:18,selectionStartColumn:1,selectionStartLineNumber:1,startColumn:1,startLineNumber:1),source:'pub+fn+div1(num:+usize)+-%3E+usize+%7B%0A++++num+/+8%0A%7D%0A%0Apub+fn+div2(num:+usize)+-%3E+usize+%7B%0A++++//+this+one+doesn!'t+even+appear+as+it!'s+just+aliased+to+div1%0A++++num+%3E%3E+3%0A%7D%0A%0Apub+fn+div3(num:+i32)+-%3E+i32+%7B%0A++++num+/+8%0A%7D%0A%0Apub+fn+div4(num:+i32)+-%3E+i32+%7B%0A++++num+%3E%3E+3%0A%7D%0A%0A'),l:'5',n:'0',o:'Rust+source+%231',t:'0')),k:50,l:'4',n:'0',o:'',s:0,t:'0'),(g:!((h:compiler,i:(compiler:r1500,filters:(b:'0',binary:'1',commentOnly:'0',demangle:'0',directives:'0',execute:'1',intel:'0',libraryCode:'0',trim:'1'),fontScale:14,j:1,lang:rust,libs:!(),options:'-O',selection:(endColumn:1,endLineNumber:1,positionColumn:1,positionLineNumber:1,selectionStartColumn:1,selectionStartLineNumber:1,startColumn:1,startLineNumber:1),source:1),l:'5',n:'0',o:'rustc+1.50.0+(Editor+%231,+Compiler+%231)+Rust',t:'0')),k:50,l:'4',n:'0',o:'',s:0,t:'0')),l:'2',n:'0',o:'',t:'0')),version:4)

**Review by richardkiss — APPROVED** [review]
Looks good, go ahead & merge after you fix the conflicts.

**Pull request merged**